apps/web/components/Embed.tsx

```diff
@@ -726,6 +769,274 @@ const ChooseEmbedTypesDialogContent = () => {
+    eventSlug: eventType?.slug ?? "",
+    eventId: eventType?.id,
+    layout: BookerLayouts.MONTH_VIEW,
+  });
+
+  const [selectedDate] = useBookerStore((state) => [state.selectedDate], shallow);
+
+  return (
+    <div className="flex flex-col">
+      <div className="mb-[9px] font-medium">
+        <Collapsible open>
+          <CollapsibleContent>
+            <div className="text-default text-sm">{t("select_date")}</div>
+            <DatePicker eventSlug={eventType?.slug} />
+          </CollapsibleContent>
+        </Collapsible>
+      </div>
+      {selectedDate ? (
+        <div className="mt-[9px] font-medium ">
+          <Collapsible open>
+            <CollapsibleContent>
+              <div
+                className="text-default mb-[9px] flex cursor-pointer items-center justify-between text-sm"
+                onClick={() => setSelectTime((prev) => !prev)}>
+                <p>{t("select_time")}</p>{" "}
+                <>
+                  {!selectedDate || !selectTime ? <ArrowDown className="w-4" /> : <ArrowUp className="w-4" />}
+                </>
+              </div>
+              {selectTime && selectedDate ? <AvailableTimeSlots eventSlug={eventType?.slug} /> : null}
+            </CollapsibleContent>
+          </Collapsible>
+        </div>
+      ) : null}
+      <div className="mb-[9px] font-medium ">
+        <Collapsible open>
+          <CollapsibleContent>
+            <div className="text-default mb-[9px] text-sm">{t("duration")}</div>
+            <TextField
+              disabled
+              label={t("duration")}
+              defaultValue={eventType?.length ?? 15}
+              addOnSuffix={<>{t("minutes")}</>}
+            />
+          </CollapsibleContent>
+        </Collapsible>
+      </div>
+      <div className="mb-[9px] font-medium ">
+        <Collapsible open>
+          <CollapsibleContent>
+            <div className="text-default mb-[9px] text-sm">{t("timezone")}</div>
+            <TimezoneSelect id="timezone" value={timezone} isDisabled />
+          </CollapsibleContent>
+        </Collapsible>
+      </div>
+    </div>
+  );
+};
+
+const EmailEmbedPreview = ({
+  eventType,
+  emailContentRef,
+  username,
+  month,
+  selectedDateAndTime,
+}: {
+  eventType: any;
+  timezone?: string;
+  emailContentRef: RefObject<HTMLDivElement>;
+  username?: string;
+  month?: string;
+  selectedDateAndTime: { [key: string]: string[] };
+}) => {
+  const [timeFormat, timezone] = useTimePreferences((state) => [state.timeFormat, state.timezone]);
+
+  return (
+    <div className="flex h-full items-center justify-center border last:font-medium">
+      <div className="m-5 max-h-full border bg-white p-4">
+        <div
+          style={{
+            paddingBottom: "3px",
+            fontSize: "13px",
+            color: "black",
+            lineHeight: "1.4",
+            minWidth: "30vw",
+            maxHeight: "60vh",
+            overflowY: "auto",
+            backgroundColor: "white",
```

Comment:
Have used inline styles here as when copying the data from this div we want the style to stay intact and in copying the content the styles need to be part of the html element rather than being computed